src/cli/schemas/oradewrc-schema.json

```diff
@@ -59,7 +59,7 @@
       ]
     },
     "compile.force": {
-      "default": true,
+      "default": false,
```

Comment:
Actually it used to be `false` by default but then I changed it because for average user I thought it is not useful if he/she works on local DB. If you compile a package specification it also invalidate package body (different timestamp) even though the content hasn't changed, you then get a warning about object change....